crates/grpc/src/protos/types/types.proto

```diff
@@ -26,6 +26,21 @@ message SupportedEntryPoint {
 }
 
 message UserOperation {
+    types.H256 hash = 1;
+    types.H160 sender = 2;
```

Comment:
I am curious why the rest is not using `UserOperationSigned` .

Reply:
Not sure what exactly you mean?

Reply:
I meant somthing like 
```
message UserOperation {
    types.H256 hash = 1;
    UserOperationSigned uo = 2;
}
```